src/common/controllers/ads-controller.js

```diff
@@ -308,6 +317,10 @@ class AdsController extends FakeEventTarget implements IAdsController {
     }
   }
 
+  get isAdPlaying(): boolean {
```

Comment:
I think isPlaying is better casue you're in adsController

Reply:
All the api of ads controller has ad inside... Rather make it consistent

Reply:
Correct I missed it, I think it should be without but it's not related to this change 👍